docs/install/ubuntu_setup.md

```diff
@@ -1,67 +1,103 @@
+First, download and install [CUDA toolkit](https://developer.nvidia.com/cuda-toolkit). CUDA 9.2 is recommended.
 
-Unzip the file and change to the cudnn root directory. Move the header and libraries to your local CUDA Toolkit folder:
+Then download [cuDNN 7.1.4](https://developer.nvidia.com/cudnn).
+
+Unzip the file and change to the cuDNN root directory. Move the header and libraries to your local CUDA Toolkit folder:
 
 ```bash
-    tar xvzf cudnn-8.0-linux-x64-v6.0.tgz
+    tar xvzf cudnn-9.2-linux-x64-v7.1
     sudo cp -P cuda/include/cudnn.h /usr/local/cuda/include
     sudo cp -P cuda/lib64/libcudnn* /usr/local/cuda/lib64
     sudo chmod a+r /usr/local/cuda/include/cudnn.h /usr/local/cuda/lib64/libcudnn*
     sudo ldconfig
 ```
 
-Finally, add configurations to config.mk file:
+<hr>
 
-```bash
-    cp make/config.mk .
-```
 
 ## Quick Installation
 
-### Install MXNet for R
+### Install MXNet for Python
 
-MXNet requires R-version to be 3.2.0 and above. If you are running an earlier version of R, run below commands to update your R version, before running the installation script.
+#### Dependencies
 
-```bash
-    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
-    sudo add-apt-repository ppa:marutter/rdev
+The following scripts will install Ubuntu 16.04 dependencies for MXNet Python development.
 
-    sudo apt-get update
-    sudo apt-get upgrade
-    sudo apt-get install r-base r-base-dev
+```bash
+wget https://raw.githubusercontent.com/apache/incubator-mxnet/master/ci/docker/install/ubuntu_core.sh
+wget https://raw.githubusercontent.com/apache/incubator-mxnet/master/ci/docker/install/ubuntu_python.sh
+sudo ./ubuntu_core.sh
+sudo ./ubuntu_python.sh
 ```
 
-To install MXNet for R:
+Using the latest MXNet with CUDA 9.2 package is recommended for the fastest training speeds with MXNet.
 
+**Recommended for training:**
 ```bash
-    # Clone mxnet repository. In terminal, run the commands WITHOUT "sudo"
-    git clone https://github.com/dmlc/mxnet.git ~/mxnet --recursive
-
-    cd ~/mxnet
-    cp make/config.mk .
-    # If building with GPU, add configurations to config.mk file:
-    echo "USE_CUDA=1" >>config.mk
-    echo "USE_CUDA_PATH=/usr/local/cuda" >>config.mk
-    echo "USE_CUDNN=1" >>config.mk
+pip install mxnet-cu92
+```
 
-    cd ~/mxnet/setup-utils
-    bash install-mxnet-ubuntu-r.sh
+**Recommended for inference:**
+```bash
+pip install mxnet-cu92mkl
 ```
-The installation script to install MXNet for R can be found [here](https://raw.githubusercontent.com/dmlc/mxnet/master/setup-utils/install-mxnet-ubuntu-r.sh).
+
+Alternatively, you can use the table below to select the package that suits your purpose.
+
+| MXNet Version | Basic | CUDA | MKL | CUDA/MKL |
+|-|-|-|-|-|
+| Latest | mxnet | mxnet-cu92 | mxnet-mkl | mxnet-cu92mkl |
+
+
+#### pip Package Availability
+
+The following table presents the pip packages that are recommended for each version of MXNet.
```

Comment:
That's awesome!

Reply:
I was able to do this because the info is available, but scattered. I wish we had this for the other things like cuDNN and numpy and OpenCV. I notice installation problems with the latter two all of the time.